BankAPILibrary/GiniBankAPILibrary/Sources/GiniBankAPILibrary/Documents/Core/GiniBankAPI.swift

```diff
@@ -92,7 +92,7 @@ extension GiniBankAPI {
         /**
          * Creates a Gini Bank API Library to be used with a transparent proxy and a custom api access token source.
          */
-        public init(customApiDomain: String,
+        public init(customApiDomain: String = "pay-api.gini.net",
```

Comment:
[Suggestion]: @igor-gini could you please use instead of "pay-api.gini.net" -> APIDomain.default?

Reply:
Sure, done